python/py-selenium/Portfile

```diff
@@ -1,30 +1,33 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
-PortGroup       python 1.0
+PortSystem          1.0
+PortGroup           python 1.0
 
-name            py-selenium
-set real_name   selenium
-version         3.141.0
-python.versions 27
-license         Apache-2
-maintainers     {dstrubbe @dstrubbe} openmaintainer
-platforms       darwin
-description     Python language binding for Selenium Remote Control
+name                py-selenium
+set real_name       selenium
+version             3.141.0
+license             Apache-2
+maintainers         {dstrubbe @dstrubbe} openmaintainer
+platforms           darwin
 
+description         Python language bindings for Selenium WebDriver
 long_description \
-    Selenium Python Client Driver is a Python language binding for \
-    Selenium Remote Control (version 1.0 and 2.0). Currently the remote protocol, \
-    Firefox and Chrome for Selenium 2.0 are supported, as well as the \
-    Selenium 1.0 bindings. As work will progresses we'll add more "native" drivers.
+    The selenium package is used to automate web browser interaction \
+    from Python. Several browsers/drivers are supported (Firefox, \
+    Chrome, Internet Explorer), as well as the Remote protocol.
 
-homepage            https://code.google.com/p/selenium/
+homepage            https://www.seleniumhq.org/
 master_sites        pypi:s/${real_name}/
 distname            ${real_name}-${version}
+
+# Support python versions
+python.versions     27 37 38
+
 checksums           rmd160  6175bf1db6e8de270d0c34b9c58649ab1b118bc7 \
                     sha256  deaf32b60ad91a4611b98d8002757f29e6f2c2d5fcaf202e1c9ad06d6772300d \
                     size    854669
 
 if {${name} ne ${subport}} {
-    depends_build-append port:py${python.version}-setuptools
+    depends_build-append \
+                    port:py${python.version}-setuptools
 }
```

Comment:
while you're making changes anyway, please add ```livecheck.type  none``` within the ```if```-block

Reply:
also it looks like there is a missing dependency, in their setup.py it says:
```    'install_requires': ['urllib3'],```

Reply:
Done thx.